syslog-ng/main.c

```diff
@@ -166,6 +167,8 @@ version(void)
          ON_OFF_STR(SYSLOG_NG_ENABLE_TCP_WRAPPER),
          ON_OFF_STR(SYSLOG_NG_ENABLE_LINUX_CAPS),
          ON_OFF_STR(SYSLOG_NG_ENABLE_SYSTEMD));
+
+  g_free(installer_version);
```

Comment:
Please merge this with the patch that introduces the g_strdup of installer_version.

Reply:
This request was cosmetic and fixed, though github cannot detect the fix. I delete the whole comment section not to confuse others.